src/index.html

```diff
@@ -125,24 +130,70 @@ <h1 class="text-white mb-6">
     <section>
       <div class="bg-background-off-white">
         <div class="max-width-container text-center">
```

Comment:
This is a bit close to the "About" section here on all screens, so we should add some space.

```suggestion
        <div class="max-width-container text-center mt-10">
```